### ISSUE TITLE
Create 1.1.0-rc.2

### ISSUE THREAD

**Issue by samreid** [phetsims/graphing-quadratics#133]
Testing completed in https://github.com/phetsims/QA/issues/366 and we would like to patch in the changes for next RC.  The changes span 2 issues and several repos of commits.

3 commits in different repos for https://github.com/phetsims/phet-io/issues/1529
1 commit in 1 repo for https://github.com/phetsims/phet-io/issues/1531

**Comment by samreid** [comment]
We need:
https://github.com/phetsims/phet-io-wrappers/commit/035efbbfae01073c5b70447fd097e7d09f56ead3
https://github.com/phetsims/phet-io/commit/f1766f6cf2a784f7f0e7e0146f0c51bea9535a6f
https://github.com/phetsims/tandem/commit/65d98f680b81fac4338239985ad940d6768e9ad8

and 

https://github.com/phetsims/phet-io-wrappers/commit/c9696c18e57c1044baa75afd9d3244de0f6c8bb9

**Comment by samreid** [comment]
Maintenance.createPatch( 'phet-io-wrappers', 'https://github.com/phetsims/graphing-quadratics/issues/133','search')
Maintenance.createPatch( 'phet-io', 'https://github.com/phetsims/graphing-quadratics/issues/133')
Maintenance.createPatch( 'tandem', 'https://github.com/phetsims/graphing-quadratics/issues/133')
Maintenance.createPatch( 'phet-io-wrappers', 'https://github.com/phetsims/graphing-quadratics/issues/133','throttle')

**Comment by zepumph** [comment]
We were able to use the maintenance release above for only one of the two bug fixes. We tried created two patches for phet-io-wrappers, which would have likely worked, except for that both commits (https://github.com/phetsims/phet-io-wrappers/commit/035efbbfae01073c5b70447fd097e7d09f56ead3 and https://github.com/phetsims/phet-io-wrappers/commit/c9696c18e57c1044baa75afd9d3244de0f6c8bb9) touched `search.js`, and the maintenance release process was not able to cherry-pick those back to back.

So we took the "throttle" patch out of this, and then called:
`Maintenance.applyPatches`
`Maintenance.updateDependencies` 
the latter created the needed branches, and so from here we could manually complete the RC:
* checkout phet-io-wrappers:graphing-quadratics1.1
* cherry-pick the "throttle" sha
* push
* update dependencies in GQ
* push
* grunt rc

Thank you @jonathanolson for discussing this with me just now, and helping us sort out why `applyPatches` was failing out.
